<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TemplateWidget</class>
 <widget class="QWidget" name="TemplateWidget">
  <property name="windowTitle">
   <string>Choose Template</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QLabel" name="templateLabel">
     <property name="text">
      <string>&amp;Template:</string>
     </property>
     <property name="buddy">
      <cstring>templateCombo</cstring>
     </property>
    </widget>
   </item>
   <item>
    <widget class="ComboBox" name="templateCombo">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="templateChooseButton">
     <property name="toolTip">
      <string>Select template file</string>
     </property>
     <property name="whatsThis">
      <string>&lt;p&gt;Browse to an existing template file.&lt;/p&gt;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="templateReloadButton">
     <property name="toolTip">
      <string>Reload template file</string>
     </property>
     <property name="whatsThis">
      <string>&lt;p&gt;Reload current template file.&lt;/p&gt;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="templateEditButton">
     <property name="toolTip">
      <string>Edit template file</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ComboBox</class>
   <extends>QComboBox</extends>
   <header>../common/utils/combobox.h</header>
  </customwidget>
 </customwidgets>
 <connections/>
</ui>
